include/aspect/velocity_boundary_conditions/gplates.h

```diff
@@ -403,33 +348,36 @@ namespace aspect
         Tensor<1,2> pointtwo;
 
         /**
-         * Determines the width of the velocity interpolation zone around the
-         * current point. Currently equals the arc distance between evaluation
-         * point and velocity data point that is still included in the
-         * interpolation. The weighting of the points currently only accounts
-         * for the surface area a single data point is covering ('moving
-         * window' interpolation without distance weighting).
+         * Determines the depth of the lithosphere. The user might want to apply
+         * the GPlates velocities not only at the surface of the model, but also
+         * in the whole lithosphere.
          */
-        double interpolation_width;
+        double lithosphere_thickness;
 
         /**
          * Pointer to an object that reads and processes data we get from
          * gplates files.
          */
-        std_cxx11::shared_ptr<internal::GPlatesLookup> lookup;
+        std_cxx11::shared_ptr<internal::GPlatesLookup<dim> > lookup;
+
+        /**
+         * Pointer to an object that reads and processes data we get from
+         * gplates files. This saves the previous data time step.
+         */
+        std_cxx11::shared_ptr<internal::GPlatesLookup<dim> > old_lookup;
 
         /**
          * Handles the update of the velocity data in lookup.
          */
```

Comment:
Can you document the input parameter. It is there to be able to reload both data files (tn and tn+1) if the model time step is larger than the data file time step.